### ISSUE TITLE
Formatting/Layout

### ISSUE THREAD

**Issue by tdurham86** [CSE512-15S/a3-tdurham-ajh24-chiasson#12]
Set classnames and group elements with div to make the site look a little more polished.

**Comment by andrewhill157** [comment]
I just made some improvements (hopefully) to behavior and scaling of the tree layout and the 3D model in f519edcfc11ed703f6edffc9ab748cc8384b82c5. I also moved the slider for the tree to the top of the tree. 

The only remaining things here (for now) are:
- Confirm that tree layout is working on someone else's computer
- See if can shorten down length of highlight dropdown (Tim was going to look into this)
- Display play button, etc. alongside 3D model to move everything up

**Comment by andrewhill157** [comment]
In my most recent commit, I moved some styling to styles.css. In practice, it is best to put as many style attributes in this file rather than using d3 or jquery selectors to set styles that are not going to change. For example, setting the width and height of a div, etc. I moved some of the styling code over to this file, so feel free to take a look at how the CSS is structured.

**Comment by tdurham86** [comment]
I commented on the commit, but I don't know if everyone sees those comments or not. I like the consistent styling, good work! However, I think we need to make the buttons closer together. At least on my monitor, if you add all of the highlighting controls (I've set it to allow up to four highlights) the controls start to overlap with the lineage slider.

**Comment by andrewhill157** [comment]
Is there a way to control text size for axis labels? Totally up to the group, but just thinking that the text on the axis labels is a little big for 3D plot.

**Comment by tdurham86** [comment]
Yep, feel free to play around with it. It's set at line 412 in the version I currently have checked out (36544551dd80a12da511873b4f48a74199dc0f9c), and is used in a 'fontstyle' element in the axis part of the dom (might be a x3dom thing, not sure this is settable by css).
